test/tests/types/basicTypegoose.test-d.ts

```diff
@@ -139,62 +139,62 @@ async function typeguards() {
     // isDocument
     {
       if (typegoose.isDocument(someNewDoc.refObjectId)) {
-        expectType<typegoose.DocumentType<TypeguardsClass>>(someNewDoc.refObjectId);
+        expect(someNewDoc.refObjectId).type.toBe<typegoose.DocumentType<TypeguardsClass>>();
       } else {
-        expectType<typegoose.mongoose.Types.ObjectId | undefined>(someNewDoc.refObjectId);
+        expect(someNewDoc.refObjectId).type.toBe<typegoose.mongoose.Types.ObjectId | undefined>();
       }
 
       if (typegoose.isDocument(someNewDoc.refString)) {
-        expectAssignable<typegoose.DocumentType<TypeguardsClass>>(someNewDoc.refString);
+        expect(someNewDoc.refString).type.toBeAssignableTo<typegoose.DocumentType<TypeguardsClass>>();
```

Comment:
`.toBeAssignableTo()` is a new matcher that shipped with TSTyche 2. Together with `.toBeAssignableWith()` they make is possible to check assignability both ways. Hence the source value can always stay on the left side of an assertion.